PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,23 @@
+# Purpose
+
+<Write your PR's purpose here>
+
+# API changes
+
+<Write the functions and structures you added or removed, if any>
```

Comment:
This should especially be written into the CHANGELOG, but a summary as PR description is indeed nice.